session/role_integ_test.go

```diff
@@ -152,7 +150,7 @@ func Test_Integ_Role(t *testing.T) {
 			}
 			// nolint: govet	// err does not shadow, using a new var to prevent data race.
 			_, err := bob.OpenCh(ctx, openingBalInfo, app, challengeDurSecs)
-			require.Error(t, err, "bob channel rejected by alice")
+			require.Error(t, err, "bob sending channel proposal should be rejected by alice")
```

Comment:
I would prefer to keep `should be`, because this message will be returned when the channel is not rejected. And the sentence describes what should have happened.